documentation_templates/tools.md

```diff
@@ -3,156 +3,109 @@
 
-> bio.tools citation: Ison, J. *et al.* (2015). [Tools and data services registry: a community effort to document bioinformatics resources.](http://nar.oxfordjournals.org/content/early/2015/11/03/nar.gkv1116.long) _Nucleic Acids Research_.
- doi: [10.1093/nar/gkv1116](http://dx.doi.org/10.1093/nar/gkv1116) PMID: [26538599 ](http://www.ncbi.nlm.nih.gov/pubmed/26538599)
+### Infrastructure usage and recommendations
 
-> EDAM citation: Ison, J., Kalaš, M., Jonassen, I., Bolser, D., Uludag, M., McWilliam, H., Malone, J., Lopez, R., Pettifer, S. and Rice, P. (2013). [EDAM: an ontology of bioinformatics operations, types of data and identifiers, topics and formats](http://bioinformatics.oxfordjournals.org/content/29/10/1325.full). _Bioinformatics_, **29**(10): 1325-1332.
-[![10.1093/bioinformatics/btt113](https://zenodo.org/badge/DOI/10.1093/bioinformatics/btt113.svg)](https://doi.org/10.1093/bioinformatics/btt113) PMID: [23479348](http://www.ncbi.nlm.nih.gov/pubmed/23479348) _Open access_
+> You could include:
+> + a link to installation instructions for each computational infrastructure 
+> + recommendations for use on a specific computational infrastructure
 
+> Documentation for a specific infrastructure should go into a infrastructure documentation template https://github.com/AustralianBioCommons/doc_guidelines/blob/master/infrastructure_optimisation.md
 
-|metadata field     | value  |
-|-------------------|:---------------------------------:|
-|Tool name          |                                   |
-|Description        |                                   |
-|Homepage URL       |                                   |
-|Software version(s)|                                   |
-|EDAM topic(s)      |                                   |
-|EDAM operation(s)  |                                   |
-|Maturity           |                                   |
-|Creators           |                                   |
-|License            |                                   |
-|Container          |                                   |
-|Install method     |                                   |
-|GitHub             |                                   |
-|bio.tools 	        |                                   |
-|BioContainers      |                                   | 
-|bioconda           |                                   |
 
----
+### Compute resource usage across tested infrastructures
 
-## Required (minimum) inputs / parameters
+> Table with high level compute resource usage information for standalone runs or testing of specific versions on specific computational infrastructures.
 
-> The minimum inputs required for the workflow to run.
+| Tool | Version | Sample description | Wall time | Cores | Peak RAM in GB (requested) | Drive (GB) | Compute system (e.g. Pawsey Setonix HPC, AWS) | Scheduler | Year-Month |
```

Comment:
This table sounds more reasonable for workflows, rather than standalone tools. For the most part, the info I look for when running a tool as an end user are: 

* Max threads 
* Peak ram used 
* Operating system compatibility: a must!   
* Dependencies 

In my opinion a lot of this detail should be moved to the benchmarking section below, where things like peak RAM, drive will potentially depend on input data size and source.